cms/static/js/views/pages/container.js

```diff
@@ -224,7 +225,7 @@ function($, _, Backbone, gettext, BasePage,
         },
 
         initializePasteButton() {
-            if (this.options.canEdit && !self.options.isIframeEmbed) {
+            if (this.options.canEdit && !this.options.isIframeEmbed) {
```

Comment:
Fixes a typo/bug introduced in https://github.com/openedx/edx-platform/pull/35587